config/jobs/kubernetes/sig-release/release-branch-jobs/1.19.yaml

```diff
@@ -256,6 +256,7 @@ periodics:
       - --provider=gce
       - --env=CL2_ENABLE_DNS_PROGRAMMING=true
       - --env=CL2_ENABLE_CLUSTER_OOMS_TRACKER=true
+      - --env=CL2_ENABLE_VIOLATIONS_FOR_SCHEDULING_THROUGHPUT=false
```

Comment:
this is a periodic

Reply:
Good catch.

Even despite it being a periodic it's a Kubemark test where we didn't want to enforce it at all.